frontend/app/(general)/profile/[username]/page.tsx

```
@@ -0,0 +1,299 @@
+                </div>
+
+                {/* Name */}
+                <h1 className="mb-2 text-center text-xl font-bold text-gray-900">
+                  {userData.firstName} {userData.lastName}
+                </h1>
+
+                {/* Bio */}
+                {userData.bio && (
+                  <div
+                    className="mb-6 text-center text-sm text-gray-600"
+                    dangerouslySetInnerHTML={{ __html: userData.bio }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)